examples/twittermap/web/conf/application.conf

```diff
@@ -80,6 +80,7 @@ cloudberry.register = "http://localhost:9000/admin/register"
 #cloudberry.ws = "ws://textdb.ics.uci.edu:9000/ws"
 cloudberry.ws = "ws://localhost:9000/ws"
 
+maxTextMessageSize = 5242880 # 5MB
```

Comment:
Could you put a comment above like the following?
Maximum allowable text message size of the web socket